src/pytest_memray/plugin.py

```diff
@@ -118,10 +140,28 @@ def pytest_unconfigure(self, config: Config) -> Generator[None, None, None]:
         if os.environ.get("MEMRAY_RESULT_PATH"):
             del os.environ["MEMRAY_RESULT_PATH"]
 
+    def _default_runner(
+        self, func: Callable[[], Any], results_file: Path, *args: Any, **kwargs: Any
+    ) -> Any:
+        result = None
+        with Tracker(results_file):
+            result = func()
+        return result
```

Comment:
Could this be written as:
```suggestion
        with Tracker(results_file):
            return func()
```
or is there some subtle difference I don't see?